utils/database.js

```diff
@@ -1421,9 +1421,19 @@ const populateItems = async (items) => {
  * @return {mongoose.HookNextFunction} HookNextFunction
  */
 const preUpdates = async (that, next, model) => {
-    if (that.getUpdate() && (that.getUpdate()._id || (that.getUpdate().$set && that.getUpdate().$set._id))) {
-        const errors = await model.statics.translationValidation(that.getUpdate().$set || that.getUpdate(), that);
-        return next(errors.length > 0 ? new Error(errors.join('\n')) : undefined);
+    const update = that.getUpdate();
```

Comment:
"PUT /api/v2/attribute HTTP/1.1" 500 - "http://localhost:3010/api/v2/attribute"
that.getUpdate is not a function